### PR TITLE
Liant1antian

### PR DESCRIPTION
**Pull request by liant1antian**

### CODE REVIEW

**Comment by shiqio3o** [comment]
good but lack of java doc

**Review by shiqio3o — APPROVED** [review]
good

File: src/view/UploadView.java [hunk 1]
```diff
@@ -98,15 +101,169 @@ public void actionPerformed(ActionEvent evt) {
...
+
+                    @Override
+                    public void keyReleased(KeyEvent e) {
+                    }
+                });
+
+
+
+
+
```
Comment (yuxizzz) [inline src/view/UploadView.java:280]: So in this class, there is only the methods? where is the constructor?

**Review by yuxizzz — CHANGES_REQUESTED** [review]

File: src/view/UploadView.java (hunk 1 above)
Comment (liant1antian) [reply src/view/UploadView.java:280]: I have wrote the constructor but it was hidden

**Pull request merged**